crates/test-collector/src/lib.rs

```diff
@@ -61,8 +62,18 @@ pub enum ExpectedTestResult {
     Panics(ExpectedPanicValue),
 }
 
+pub trait ForkConfig {}
+
+impl ForkConfig for ValidatedForkConfig {}
+impl ForkConfig for RawForkConfig {}
+
 #[derive(Debug, Clone, PartialEq)]
-pub enum ForkConfig {
+pub struct ValidatedForkConfig {
```

Comment:
It is used only in forge, but I would keep it in this crate for consistency

Reply:
I'd move it to forge, this structure doesn't represent something returned from test collector, it is our abstraction over what collector can return.